deps/npm/package.json

```diff
@@ -204,7 +204,7 @@
     "sprintf-js": "~1.0.3",
     "standard": "~6.0.8",
     "tacks": "~1.2.2",
-    "tap": "~9.0.3"
+    "tap": "*"
```

Comment:
This change will be overwritten when we update npm. I think it's better to not edit the files under the `dep` directory for a specific reason except deps-update.